1906-11-24.xml

```diff
@@ -7,12 +7,12 @@
             <titleStmt>
                 <title level="m" type="main">Digital Egyptian Gazette</title>
                 <title level="m" type="sub">An encoded transcription</title>
-                <editor>Adrian Ell</editor>
+                <editor>Brett Barry</editor>
```

Comment:
add another line with your name--don't remove Adrian's, as she did the original work.